README.md

```diff
@@ -141,36 +141,36 @@ Otherwise, you can simply import along with the backend itself (as shown above).
+|[`handleInputBlur`](#handleInputBlur) | `Function` | `undefined` | Event handler for input onBlur [Demo](http://www.jinno.io/app/6/handleInputBlur?source=react-tag-input)
+|[`minQueryLength`](#minQueryLength) | `Number` | `2` | How many characters are needed for suggestions to appear [Demo](http://www.jinno.io/app/6/minQueryLength?source=react-tag-input)
+|[`removeComponent`](#removeComponent) | `Function` |  | Function to render custom remove component for the tags. [Demo](http://www.jinno.io/app/6/removeComponent?source=react-tag-input)
+|[`autocomplete`](#autocomplete) | `Boolean`/`Number` | `false` | Ensure the first matching suggestion is automatically converted to a tag when a [delimiter](#delimiters) key is pressed [Demo](http://www.jinno.io/app/6/autocomplete?source=react-tag-input)
+|[`readOnly`](#readOnly) | `Boolean` | `false` | Read-only mode without the input box and `removeComponent` and drag-n-drop features disabled [Demo](http://www.jinno.io/app/6/readOnly?source=react-tag-input)
+|[`name`](#name) | `String` | `undefined` | The `name` attribute added to the input [Demo](http://www.jinno.io/app/6/name?source=react-tag-input)
+|[`id`](#id) | `String` | `undefined` | The `id` attribute added to the input [Demo](http://www.jinno.io/app/6/id?source=react-tag-input)
+|[`maxLength`](#maxLength) | `Number` | `Infinity` | The `maxLength` attribute added to the input [Demo](http://www.jinno.io/app/6/maxLength?source=react-tag-input)
+|[`inline`](#inline) | `Boolean` | `true` | Render input field and selected tags in-line [Demo](http://www.jinno.io/app/6/inline?source=react-tag-input)
 |[`inputFieldPosition`](#inputFieldPosition) | `String` | `inline` | Specify position of input field relative to tags
-|[`allowUnique`](#allowUnique) | `Boolean` | `true` | Boolean value to control whether tags should be unqiue
-|[`allowDragDrop`](#allowDragDrop) | `Boolean` | `true` | Boolean value to control whether tags should have drag-n-drop features enabled
-|[`renderSuggestion`](#renderSuggestion) | `Function` | `undefined` | Render prop for rendering your own suggestions
+|[`allowUnique`](#allowUnique) | `Boolean` | `true` | Boolean value to control whether tags should be unqiue [Demo](http://www.jinno.io/app/6/allowUnique?source=react-tag-input)
+|[`allowDragDrop`](#allowDragDrop) | `Boolean` | `true` | Boolean value to control whether tags should have drag-n-drop features enabled [Demo](http://www.jinno.io/app/6/allowDragDrop?source=react-tag-input)
+|[`renderSuggestion`](#renderSuggestion) | `Function` | `undefined` | Render prop for rendering your own suggestions [Demo](http://www.jinno.io/app/6/renderSuggestion?source=react-tag-input)
 | [`inputProps`](#inputProps) | Object |`{}` | The extra attributes which are passed to the input field.
 
-### tags
+### tags [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```

Comment:
```suggestion
### tags 
Check out the [Demo](http://www.jinno.io/app/6/tags?source=react-tag-input)
```
The demo along with header tag looks pretty big, lets add it to the next line for all as shown above

Reply:
Also demo link for [`delimeters`](https://github.com/react-tags/react-tags/blob/965e654b8b4c50cde31a92f7a23e980c15f53250/README.md#delimiters) is missing